src/qt/walletmodel.cpp

```diff
@@ -54,6 +54,20 @@ WalletModel::~WalletModel()
     unsubscribeFromCoreSignals();
 }
 
+void WalletModel::loadWalletOptions()
+{
+    // update optionsModel with wallet options
+    optionsModel->setSST(static_cast<double>(getWalletStakeSplitThreshold() / COIN));
+    optionsModel->refreshDataView();
+}
+
+void WalletModel::resetOptionsToDefault(QSettings& settings)
```

Comment:
Wouldn't be better to call it `resetWalletOptions()` ? So we know that the options are only for the wallet.

Reply:
done.